src/vmm/tests/cpu_templates.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::PathBuf;
+use vmm::cpu_config::templates::common_types::StaticCpuTemplate;
+use vmm::cpu_config::templates::{CpuTemplateType, GetCpuTemplate};
+
+fn get_json_template_file_path(filename: &str) -> PathBuf {
+    [
+        env!("CARGO_MANIFEST_DIR"),
+        "../../resources/tests/static_cpu_templates",
+        filename,
+    ]
+    .iter()
+    .collect()
+}
+
+fn verify_consistency_static_template(static_template: StaticCpuTemplate, filename: &str) {
+    let hardcoded_template = Some(CpuTemplateType::Static(static_template))
+        .get_cpu_template()
+        .unwrap();
+
+    let path = get_json_template_file_path(filename);
+    let json_template = serde_json::from_str(&std::fs::read_to_string(path))?;
+}
+
+#[test]
+#[cfg(target_arch = "x86_64")]
+fn verify_consistency_x86_static_templates() {
+    const TEMPLATES: [(StaticCpuTemplate, &str); 1] = [(StaticCpuTemplate::C3, "c3.json")];
+
+    for (template, filename) in TEMPLATES {
+        println!("{:#?}", get_json_template_file_path(filename));
```

Comment:
is this print essential?

Reply:
https://github.com/firecracker-microvm/firecracker/pull/3802#discussion_r1221300311